osu.Game.Rulesets.Osu/Skinning/LegacyOldStyleSpinner.cs

```diff
@@ -25,12 +25,16 @@ public class LegacyOldStyleSpinner : CompositeDrawable
         private Sprite metreSprite;
         private Container metre;
 
+        private bool spinnerBlink;
+
         private const float sprite_scale = 1 / 1.6f;
         private const float final_metre_height = 692 * sprite_scale;
 
         [BackgroundDependencyLoader]
         private void load(ISkinSource source, DrawableHitObject drawableObject)
         {
+            spinnerBlink = !source.GetConfig<OsuSkinConfiguration, bool>(OsuSkinConfiguration.SpinnerNoBlink)?.Value ?? true;
```

Comment:
Not sure if it's just a "me" problem, but this makes my brain hurt. I don't immediately know reading this if the default is `false` or `true` as I don't remember operator precedence by heart and I have to add braces to check (it does seem to be correct, however).

Reply:
Thanks! Added braces